integration-tests/lts/select.test.ts

```diff
@@ -310,14 +310,15 @@ describe("select", () => {
   test("* in polymorphic", async () => {
     const q = e.select(e.Person, () => ({
       ...e.is(e.Hero, e.Hero["*"]),
+      name: true,
```

Comment:
This change reflects the new behavior of 4.x where the parent type's cardinality is considered when using polymorphic queries. In this case, `Person` has a required `name`, but the query can return `Person` records that are not `Hero`s, which would return a `null` name, which is invalid for `Person`.